cloud/monitoring.rst

```diff
@@ -80,6 +80,120 @@ Failovers happen to address hardware failure, as mentioned, and also for other r
 *  Situation
 *  Follower
 
+StatsD external reporting
+=========================
+
+Citus Cloud can send events to an external `StatsD <https://github.com/etsy/statsd>`_ server for detailed monitoring. Cloud sends the following statsd metrics:
+
++---------------------------------------------+------------------------------------+
+| Metric                                      | Notes                              |
++=============================================+====================================+
+| citus.disk.data.total                       |                                    |
++---------------------------------------------+------------------------------------+
+| citus.disk.data.used                        |                                    |
++---------------------------------------------+------------------------------------+
+| citus.load.1                                | Load in past 1 minute              |
++---------------------------------------------+------------------------------------+
+| citus.load.5                                | Load in past 5 minutes             |
++---------------------------------------------+------------------------------------+
+| citus.load.15                               | Load in past 15 minutes            |
++---------------------------------------------+------------------------------------+
+| citus.locks.granted.<mode>.<locktype>.count | See below                          |
++---------------------------------------------+------------------------------------+
+| citus.mem.available                         |                                    |
++---------------------------------------------+------------------------------------+
+| citus.mem.buffered                          |                                    |
++---------------------------------------------+------------------------------------+
+| citus.mem.cached                            |                                    |
++---------------------------------------------+------------------------------------+
+| citus.mem.commit_limit                      | Memory currently available to      |
+|                                             | be allocated on the system         |
```

Comment:
Might be worth mentioning that these come from /proc/meminfo, so if one wants to understand more they can look there.

Reply:
Added a note about that below this table.